src/main/webapp/assets/modules/navigation/navigation.con.js

```diff
@@ -1,6 +1,26 @@
-angular.module("Navigation").controller("navigationController", function ($scope, securityService) {
-
+techlooper.controller("navigationController", function ($scope, securityService, apiService, localStorageService, $location,
+                                                  jsonValue, utils, $timeout, $rootScope) {
```

Comment:
Line is too long.<br>'$rootScope' is defined but never used.<br>'$timeout' is defined but never used.